js/ui/date_box/ui.date_box.js

```diff
@@ -295,6 +295,12 @@ var DateBox = DropDownEditor.inherit({
             {
                 device: { platform: "ios" },
                 options: {
+                    /**
+                    * @name dxDateBoxOptions_pickerType
+                    * @publicName pickerType
+                    * @default 'native'
```

Comment:
In doc comment, the platform is not mentioned
@default 'native' @for iOS 
same for sndroid